### PR TITLE
simple node server with very minimal packages such as cors and helmet

### PR DESCRIPTION
**Pull request by thinklikeadesigner**
created server with the following 

```javascript
const cors = require('cors');
require('dotenv').config();

const express = require('express');
const mongoose = require('mongoose');
const helmet = require('helmet');
const { errors } = require('celebrate');
```

### CODE REVIEW

File: backend/app.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
+const cors = require('cors');
+require('dotenv').config();
+
+const express = require('express');
+const mongoose = require('mongoose');
+const helmet = require('helmet');
+const { errors } = require('celebrate');
+
+
```
Comment (vgalilei) [inline backend/app.js:9]: Do we need this extra spaces ?

File: backend/app.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
+const cors = require('cors');
```
Comment (McCambley) [inline backend/app.js:1]: missing cors and dotenv in package.json yeah?

**Review by McCambley — CHANGES_REQUESTED** [review]
Nice Rebecca! 
I think it's missing a few dependencies, but other than that it looks good. If you add those I can test it then approce it for merging.

**Comment by thinklikeadesigner** [comment]
Which ones? I was going to add the rate limiter, logger, etc etc in a different commit. I'm really bad at this "small PR" thing

**Comment by thinklikeadesigner** [comment]
Oh, the actual package.json dependencies. Cool, I added them in my latest commit.

File: backend/app.js (hunk 2 above)
Comment (thinklikeadesigner) [reply backend/app.js:1]: done

File: backend/app.js (hunk 1 above)
Comment (thinklikeadesigner) [reply backend/app.js:9]: not really, fixed it

**Review by idenispankov — APPROVED** [review]
LGTM

**Comment by thinklikeadesigner** [comment]
> Nice Rebecca! I think it's missing a few dependencies, but other than that it looks good. If you add those I can test it then approce it for merging.

done!

**Pull request merged**